### PR TITLE
Add Chris' Gilbertsville file to Master

### LINKED ISSUE #13: Create a file for cshing1 Gilbertsville, PA
- [ ] Create a branch
- [ ] Adding commits
- [ ] Open a Pull Request
- [ ] Collaborate
- [ ] Receive Approval :+1: 
- [ ] Merge :shipit:

### PR DESCRIPTION
**Pull request by cshing1**
closes #13

### CODE REVIEW

File: cshing1-Gilbertsville.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Gilbertsville, PA
+
+#### Places to Go
+- Zerns Farmer's Market
+
+#### Places to Eat
+- Shorty's Cafe
+- Shady Maple
```
Comment (praduns) [inline cshing1-Gilbertsville.md:8]: Is this a pancake place? :laughing:

**Comment by cshing1** [comment]
@praduns Does this pull request look good to :shipit: ?

**Comment by githubteacher** [comment]
👍

**Comment by praduns** [comment]
Looks good :+1:

**Pull request merged**